contrib/inventory/nsot.py

```diff
@@ -0,0 +1,345 @@
+#!/bin/env python2.7
```

Comment:
you might want to make it python vs python2.7 which will not work on all platforms